### PR TITLE
polish(zhangpai): polish slime_volley league training

### PR DESCRIPTION
**Pull request by zhangpaipai**
## Description
图片
图片

## Related Issue
#90 

## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

**Comment by zxzzz0** [comment]
Please plot a curve of TrueSkill score vs training time.

File: ding/league/shared_payoff.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def _win_rate(self, home: str, away: str) -> float:
         handle = self._data[key]
         # No enough game records.
         if handle['games'] < self._min_win_rate_games:
-            return 0.5
+            return 0.1
```
[the remark this answers is not in the document]
Comment (zhangpaipai) [reply ding/league/shared_payoff.py:142]: before finding that snapshot bug, i thought 100% win rate was caused by the reason that main player was hard to choose just snapshot player and got stronger with choosing other players. now i change back to 0.5.

**Comment by zxzzz0** [comment]
What is the latest main agent's mean reward when playing with [this AI](https://blog.otoro.net/2015/04/05/levelling-the-playing-field/)?

**Comment by zxzzz0** [comment]
@PaParaZz1 Why this Pull Request is closed?

**Pull request closed without merging**